src/app/shared/components/template/components/date-time-picker/date-time-picker.component.ts

```diff
@@ -0,0 +1,41 @@
+import { Component, computed, OnInit, signal, WritableSignal } from "@angular/core";
+import { TemplateBaseComponent } from "../base";
+import { getStringParamFromTemplateRow, parseBoolean } from "src/app/shared/utils";
+
+interface IDateTimePickerParams {
+  /** TEMPLATE PARAMETER: "disabled". If true, date time picker is disabled and greyed out */
+  disabled: boolean;
+  /** TEMPLATE PARAMETER: "type". Determines the type of picker date & time ('datetime'), date only ('date') or time only ('time') picker*/
+  type: string;
+}
+
+@Component({
+  selector: "plh-date-time-picker",
+  templateUrl: "./date-time-picker.component.html",
+  styleUrls: ["./date-time-picker.component.scss"],
+})
+export class TmplDateTimePickerComponent extends TemplateBaseComponent implements OnInit {
+  public dateTimeValue: WritableSignal<string | undefined>;
+  public params = computed(() => this.getParams());
+
+  constructor() {
+    super();
+  }
+
+  public ngOnInit(): void {
+    this.dateTimeValue = signal(this.value() || undefined);
+  }
+
+  public handleChange(value: string | string[]) {
+    this.setValue(value);
+  }
+
+  private getParams(): IDateTimePickerParams {
+    let parameterList = this.parameterList();
+
+    return {
+      disabled: parseBoolean(parameterList.disabled),
+      type: getStringParamFromTemplateRow(this._row, "type", "date-time"),
```

Comment:
nit(non-blocking): a lot of these `getParamFromTemplate` style utility functions are a bit clunky/messy, and more designed for an earlier time where we struggled a lot with passing different data types between components. I think if creating a new component it might be better to define in simpler terms where possible (like you've done for the disabled parameter), 

i.e. just using parameter list with fallback value (using logical OR)
```ts
type: parameterList.type || 'date-time'
```